README.md

```diff
@@ -31,11 +31,26 @@ support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testi
 * [2024/07] Performance optimizations and usability improvements on [client-side](https://github.com/intel/neural-compressor/blob/master/docs/source/3x/client_quant.md).
 
 ## Installation
+### Install Framework
+#### Install torch for CPU
+```Shell
+pip install torch --index-url https://download.pytorch.org/whl/cpu
+```
+#### Install torch/intel_extension_for_pytorch for XPU
```

Comment:
XPU may not be clear to external user,  how about using "Intel GPU"

Reply:
ok